python/paddle/fluid/tests/unittests/CMakeLists.txt

```diff
@@ -82,12 +82,14 @@ if(NOT WITH_GPU OR WIN32)
 endif()
 
 if(WIN32)
-    LIST(REMOVE_ITEM TEST_OPS test_boxps)
-    LIST(REMOVE_ITEM TEST_OPS test_trainer_desc)
     LIST(REMOVE_ITEM TEST_OPS test_multiprocess_reader_exception)
+endif()
+
+if(NOT WITH_DISTRIBUTE OR WIN32)
```

Comment:
Some unittests is under WITH_DISTRIBUTE.

Reply:
done, thx